modular_citadel/code/modules/reagents/objects/items.dm

```diff
@@ -0,0 +1,105 @@
+/obj/item/FermiChem/pHbooklet
+    name = "pH indicator booklet"
+    desc = "A booklet containing paper soaked in universal indicator."
+    icon_state = "pHbooklet"
```

Comment:
by the by the icon for this is way too large.

Reply:
Are you sure? This is their size https://puu.sh/DFeXO.png